pkg/operator/starter.go

```diff
@@ -9,6 +9,8 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/mfojtik/bugzilla-operator/pkg/operator/escalationcontroller"
```

Comment:
nit the imports seem to be a bit disordered. there a couple of separate local package imports. Can we group them with the ones at the end?